### PR TITLE
Add DRM handle

### PR DESCRIPTION
**Pull request by StratusFearMe21**
See [my PR for winit](https://github.com/rust-windowing/winit/pull/2272)

### CODE REVIEW

**Comment by Lokathor** [comment]
I'm very happy to see this added!

Is it all set to be merged or did you want to do more?

File: src/unix.rs [hunk 1]
```diff
@@ -58,6 +58,13 @@ pub struct WaylandHandle {
     pub display: *mut c_void,
 }
 
+#[non_exhaustive]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+pub struct DrmHandle {
```
Comment (madsmtm) [inline src/unix.rs:63]: Missing documentation.

File: src/lib.rs [hunk 2]
```diff
@@ -160,4 +160,9 @@ pub enum RawWindowHandle {
     /// ## Availability Hints
     /// This variant is used on HaikuOS.
     Haiku(HaikuHandle),
+    /// A raw window handle for the Linux Kernel Mode Set/Direct Rendering Manager
+    ///
+    /// ## Availability Hints
+    /// This variant is used on Linux when X nor Wayland are available
+    Drm(DrmHandle),
```
Comment (madsmtm) [inline src/lib.rs:167]: Maybe place this below the `Wayland` variant?

**Review by madsmtm — CHANGES_REQUESTED** [review]

File: src/lib.rs (hunk 2 above)
Comment (StratusFearMe21) [reply src/lib.rs]: Awesome, done

File: src/unix.rs (hunk 1 above)
Comment (StratusFearMe21) [reply src/unix.rs:71]: Whoops, sorry about that, just fixed it

**Comment by StratusFearMe21** [comment]
It is all set to be merged

**Review by Lokathor — APPROVED** [review]

**Comment by Lokathor** [comment]
I'm happy with this, but I want to give @madsmtm a chance to double check.

File: src/lib.rs [hunk 3]
```diff
@@ -135,6 +135,11 @@ pub enum RawWindowHandle {
     /// This variant should be expected anywhere Wayland works, which is
     /// currently some subset of unix systems.
     Wayland(WaylandHandle),
+    /// A raw window handle for the Linux Kernel Mode Set/Direct Rendering Manager
+    ///
+    /// ## Availability Hints
+    /// This variant is used on Linux when X nor Wayland are available
```
Comment (madsmtm) [inline src/lib.rs:141]: Nit: You committed my `neither` suggestion, but I think it got lost in a merge

**Review by madsmtm — APPROVED** [review]
My blocker was having no docs on `DrmHandle`, with that fixed I'm satisfied

**Pull request merged**